front/components/vaults/VaultMembers.tsx

```diff
@@ -33,6 +33,15 @@ const tableColumns = [
       </DataTable.CellContent>
     ),
     accessorFn: (row: RowData) => row.name,
+    filterFn: (row: Row<RowData>, columnId: string, filterValue: any) => {
+      if (!filterValue) {
+        return true;
+      }
+
+      return removeDiacritics(row.original.name)
```

Comment:
NIT: Same here, I think you can do `row.getValue("name")`

Reply:
thanks ! I was looking for that to make the function easily re-usable.